### PR TITLE
Mailet 122 - CI 

### PR DESCRIPTION
**Pull request by chibenwa**
https://github.com/apache/james-project/pull/61

### CODE REVIEW

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
test this please

File: mailet/standard/src/main/java/org/apache/james/transport/matchers/RecipientIs.java [hunk 1]
```diff
@@ -17,29 +17,29 @@
...
     public void init() throws javax.mail.MessagingException {
-        StringTokenizer st = new StringTokenizer(getCondition(), ", \t", false);
-        recipients = new java.util.HashSet<MailAddress>();
-        while (st.hasMoreTokens()) {
-            recipients.add(new MailAddress(st.nextToken()));
+        if (Strings.isNullOrEmpty(getCondition())) {
+            throw new MessagingException("RecipientIs should have a condition  composed of a list of mail addresses");
+        }
+        recipients = MailAddressCollectionReader.read(getCondition());
+        if (recipients.size() < 1) {
```
Comment (rouazana) [inline mailet/standard/src/main/java/org/apache/james/transport/matchers/RecipientIs.java]: recipients.isEmpty()?

File: mailet/standard/src/main/java/org/apache/james/transport/matchers/UserIs.java [hunk 2]
```diff
@@ -17,39 +17,39 @@
...
-            users.add(st.nextToken());
+
+    Set<String> users;
+
+    public void init() throws MessagingException {
+        if (Strings.isNullOrEmpty(getCondition())) {
+            throw new MessagingException("UserIs should have a condition composed of a list of local parts of mail addresses");
+        }
+        users = ImmutableSet.copyOf(Splitter.on(", ").split(getCondition()));
+        if (users.size() < 1) {
```
Comment (rouazana) [inline mailet/standard/src/main/java/org/apache/james/transport/matchers/UserIs.java]: recipients.isEmpty()?

**Pull request merged**